src/styles/blocks/stars.css

```diff
@@ -0,0 +1,41 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url('../images/star.svg');
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
```

Comment:
The selector '.stars--0 .stars__star:nth-child(-n + 0)' is redundant because '-n + 0' will not select any elements. Consider removing this rule if it is not needed.